content/concepts/content-management/02-content-models.md

```diff
@@ -6,20 +6,47 @@ metaDescription: "sensenet Content Model and Schema"
 
 sensenet organizes content into content repositories, that allows you to group all the related content for a project together. Since everything is a content is sensenet this repository includes documents, workspaces, users, settings, workspaces and many more.
```

Comment:
```suggestion
sensenet organizes content into content repositories, that allows you to group all the related content for a project together. Since everything is a content in sensenet, this repository includes documents, workspaces, users, settings, workspaces and many more.
```